universal-login-wallet/test/integration/ui/CreateAccount.tsx

```diff
@@ -24,6 +24,7 @@ describe('UI: Creation flow', () => {
         ({relayer, provider} = await setupSdk(wallet, '33113'));
         services = await createPreconfiguredServices(provider, relayer, [ETHER_NATIVE_TOKEN.address]);
         await services.sdk.tokensDetailsStore.fetchTokensDetails();
+        await services.sdk.fetchRelayerConfig();
```

Comment:
I think that we should move these two lines to `createPreconfiguredServices`. 
We can also run just `await sdk.start()`